debian-9/init_container.sh

```diff
@@ -20,11 +20,55 @@ sed -i "s/SSH_PORT/$SSH_PORT/g" /etc/ssh/sshd_config
 /usr/sbin/sshd
 
 appPath="/home/site/wwwroot/"
+userStartupCommand="$@"
```

Comment:
@patricklee2 this duplication is costly as it is easy to fix in one place and not the other. Both the files' contents are the same.